src/sbuf/SBuf.cc

```diff
@@ -21,54 +21,60 @@
 
 InstanceIdDefinitions(SBuf, "SBuf");
 
-SBufStats SBuf::stats;
 const SBuf::size_type SBuf::npos;
 const SBuf::size_type SBuf::maxSize;
 
+SBufStats &
+SBuf::Stats()
+{
+    static auto *stats = new SBufStats;
```

Comment:
Please apply NoNewGlobals code pattern rather than using a similar but different one:

```suggestion
auto &
SBuf::Stats()
{
    static const auto stats = new SBufStats();
```

N.B. The same applies to virtually any methods in other PRs that can use that pattern. Reusing that pattern helps improve code quality and reduce review iterations/delays.